### ISSUE TITLE
Documentation does not show usage within `<scripts>` - How to use `$t`, `$tc`, `$d`, `$n`?

### ISSUE THREAD

**Issue by MickL** [intlify/vue-i18n#1733]
### Clear and concise description of the problem

The docs are really great! I just dont see how to use all this amazing template functions within the script, e.g. `$t`, `$tc`, `$d`, `$n`. If I try to use them within <scripts> I get an error that they are not defined.

### Suggested solution

For each template function provide one example on how to use them within <script>

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

P.S. When googling "vue i18n" the first result links to 8.x docs with no information that this is an old version. This is especially problematic because there are breaking changes and also the new docs look more nice :) It would be great to add a message that there is a more current version. E.g. Bootstrap adds an info at the very top: https://getbootstrap.com/docs/4.6/getting-started/introduction/

**Comment by HendrikJan** [comment]
Do you know how to do it?

I'm looking for an answer as well.

**Comment by HendrikJan** [comment]
Okay, than I'm having another problem.
I'm using Nuxt3 as well.

`useI18n()` tells me: "cannot find name 'useI18n'".

But if I do `import { useI18n } from 'vue-i18n'` is tells me: "'vue-i18n' has no exported member 'useI18n`"

It seems like this is no problem in your case.

**Comment by MickL** [comment]
Yes it is. But regarding your issue: You will not import it from vue-18n because it is from nuxt-i18n. But it is already auto imported. My guess is you didnt add `@nuxtjs/i18n` to your nuxt.config modules

**Comment by HendrikJan** [comment]
True, because the documentation of v9 did not tell met in install `@nuxtjs/i18n`.
It tells to write your own Nuxt plugin: https://vue-i18n.intlify.dev/guide/integrations/nuxt3.html

I'll see if I can install and use `@nuxtjs/i18n`.

**Comment by MickL** [comment]
Use `@nuxtjs/i18n` instead, it is pretty awesome!

**Comment by MickL** [comment]
After two years of use, I dont destructure anymore in my Nuxt projects. Instead, everything is availabile without any import, e.g. `$n(100, 'currency');`. Not sure if this is a Nuxt thing tho.

**Comment by BobbieGoede** [comment]
@MickL that's a [nuxt-i18n feature](https://i18n.nuxtjs.org/docs/api/options#autodeclare), I'm going to extract that into a separate vite plugin someday so vue-i18n users can use it in their projects as well.

**Comment by BobbieGoede** [comment]
I have extracted the nuxt-i18n feature into an vite/unplugin so projects using vue-i18n directly can use it too. 

The plugin injects `const { t: $t, n: $n, ... } = useI18n()` whenever `$t` or other functions are used in a component script setup, mimicking the global functions available in the template. This behavior relies on auto-imports to automatically import the composable.

Still alpha versioning but eventually nuxt-i18n will use the plugin as well, you can find it here https://github.com/BobbieGoede/unplugin-auto-declare
